dist/rpm/openQA.spec

```diff
@@ -290,6 +290,10 @@ rm \
 # within CI systems, e.g. OBS. See t/lib/OpenQA/Test/TimeLimit.pm
 export CI=1
 export OPENQA_TEST_TIMEOUT_SCALE_CI=10
+# Skip container tests that would need additional requirements, e.g.
+# docker-compose. Also, these tests are less relevant (or not relevant) for
+# packaging
+export CONTAINER_TEST=0
```

Comment:
I would've said "tests we already run via GHA" 🤔 what is less relevant?

Reply:
Well, my point was that building a container is actually a different use case than installing packages. Of course we use RPM packages within the containers but that's already tested itself as well.